docs/source/intro.rst

```diff
@@ -11,20 +11,20 @@ Don't worry if you've not seen, or never even heard of Python_, it's incredibly
 To whet your appetite, here's a quick (complete) sample script that shows some of the power of Python Script.::
    
    editor.replace("old", "new")
-   editor.pyreplace(r"^Code: ([A-Z]{4,8})", r"The code is \1")
-
-   notepad.runMenuCommand("TextFX Tools", "Delete Blank Lines")
+   
+   notepad.runMenuCommand("Line Operations", "Remove Empty Lines")
+   
    notepad.save()
 
 
-Line 1 performs a normal search and replace on the current document, replacing the word "old" with the word "new".
-
-Line 2 also performs a search and replace, but uses Python_ regular expressions, which is a much more complete implementation of regular expressions than is natively supported in Notepad++. 
+Line 1 performs a normal search and replace on the current document, replacing the word "old" with the word "new". 
 
-Line 4 runs the menu command called "Delete Blank Lines" from the TextFX Tools menu.
+Line 3 runs the menu command called "Remove Empty Lines" from menu: Edit -> Line Operations.
 
 Line 5 saves the current document (equivalent to clicking File, Save).
 
+WARNING: While for the most part, a PythonScript script will behave like a Python script run by a Python interpreter, one important difference is that ``exit()``, ``quit()`` and related functions like ``sys.exit()`` and ``os._exit(0)``, that normally terminate the script and exit the Python interpreter, in PythonScript they will also abruptly terminate Notepad++ (crash it without saving your unsaved files and other data). So, you best avoid them, and use other techniques for early exiting from your code (like ``return`` inside a function, for example). 
+
 
 Objects  
 ========

```

Comment:
@Ekopalypse , which exmple exactly? Please comment directly under the changes that you refer to, thanks.

Reply:
I guess you mean in intro.rst#working-with-unicode-text . 

Please correct me if I'm wrong: after merging this doc PR, people will get to see it in their installed PS only after PS project releases a new version.
If so, will the new PS version be already with Python 3? 

If yes, then why need to make it compatible with both py 2 and 3, instead of only py 3? 

And, also other parts of doc will need adjustments to py 3.

Reply:
Yes, the changes only apply to a new version of PS(2/3).
As far as I know, there is currently only one issue that prevents the project from deprecating the PS2 version, and that is exactly the discussion point of the example. Strings.
In Python2, a string is an array of bytes, while in Python3, a string is now a Unicode object. In Python3, a string has only one encode method to convert it to a byte array, and the bytes object knows only one decode method to create a Unicode object.
I haven't checked if other parts of the document need to be updated as well, but right now I can only think of the "print" function as a possible stumbling block.

Reply:
> I haven't checked if other parts of the document need to be updated as well, but right now I can only think of the "print" function as a possible stumbling block.

Aside from what you say, I noticed a few mentionings of py 2 or 2.7. 

If you're not sure that next PS version will be with python3, then let's just push this PR with current changes, then later can make another one specifically to ensure all is for python 3.

Reply:
Or let @chcg  weigh in